### PR TITLE
feat: implementing delete endpoint and email confirmation

### PR DESCRIPTION
**Pull request by agnbart**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task Goal
The goal of this task is to implement the deletion of the crypto alert.

### Requirements
1. Endpoint for deleting new alert must be created.
2. An email confirmation should be sent to the provided email address upon successful deletion of the alert.

### Required endpoint schema
`DELETE /alerts/:id`

### Code Review

#### src/alert/alert.controller.ts
The code changes in this file cover the task requirements. The `remove` method has been updated to handle the deletion of an alert. The endpoint `DELETE /alerts/:id` is correctly implemented.

#### src/alert/alert.service.ts
The code changes in this file also cover the task requirements. The `remove` method has been updated to delete an alert from the database using the `prismaService`. After successful deletion, an email is sent using the `mailjetService`.

However, there is a potential issue with error handling. If an error occurs during the deletion of an alert, the error message is set to 'Error on deleting Alert.' but the error itself is not logged or returned. This could make debugging difficult if an error occurs. It would be better to include the actual error message in the response.

Here's an example of how you could improve this:

```diff
.catch((error) => {
-  errorMsg = 'Error on deleting Alert.';
+  errorMsg = `Error on deleting Alert: ${error.message}`;
});
```

This way, you will have more information about what went wrong if an error occurs.

In conclusion, the code changes cover the task requirements, but there is room for improvement in error handling.
